pkg/storage/client_raft_test.go

```diff
@@ -1622,18 +1622,8 @@ func TestReportUnreachableHeartbeats(t *testing.T) {
 	mtc := startMultiTestContext(t, 3)
 	defer mtc.Stop()
 
-	// Replicate the range onto all three stores
 	mtc.replicateRange(1, 1, 2)
 
-	// Send a single increment to ensure the range is up.
```

Comment:
This line seems to be making sure that the range is present on all the intended replicas, which is done in https://github.com/cockroachdb/cockroach/blob/4526f02/pkg/storage/client_test.go#L931. I originally thought that was added in https://github.com/cockroachdb/cockroach/pull/10634/commits/eca090bf6c5875eb4249db6884af5869558ebab5#diff-43afc4d5ffd4c8f3180911452dc71ce8R928, but it was always there.

looking at this again, I don't know why this was ever here to begin with. do you? the test continues to pass under stress.